### PR TITLE
UNITY-8284 create-variation-products

### PR DESCRIPTION
**Pull request by mhoehn**

### CODE REVIEW

File: apps/data-types.md [hunk 1]
```diff
@@ -933,6 +933,28 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | link | [link](page:apps-data-types#link)| The link to the product variation. |
 | attributeSelection | array of [attributeSelection](page:apps-data-types#attributeselection) | The attribute of the selected product variation. |
 
+## variation (create request)
+
+| Attribute | Type | Description |
+| - | - |  - |
+| productType | string | Unique name (alias) of the product type. |
+| visibleAfterCreation | boolean | Indicates if the new variation products are displayed in the shop after creation. |
```
Comment (dkluever22) [inline apps/data-types.md:941]:
```suggestion
| visibleAfterCreation | boolean | Indicates if the new product variations are displayed in the shop after creation. |
```

File: apps/data-types.md [hunk 2]
```diff
@@ -933,6 +933,28 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | link | [link](page:apps-data-types#link)| The link to the product variation. |
 | attributeSelection | array of [attributeSelection](page:apps-data-types#attributeselection) | The attribute of the selected product variation. |
 
+## variation (create request)
+
+| Attribute | Type | Description |
+| - | - |  - |
+| productType | string | Unique name (alias) of the product type. |
```
Comment (dkluever22) [inline apps/data-types.md:940]:
```suggestion
| productType | string | The unique name of the product type. |
```

**Review by dkluever22 — CHANGES_REQUESTED** [review]
@mhoehn I have reviewed the PR and left you a couple of comments. Please take a look and let me know if you have any questions :)

File: _raml/apps/products/examples/post_product_productid_variations_request.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{
+  "productType":"Shoe",
```
[the remark this answers is not in the document]
Comment (mhoehn) [reply _raml/apps/products/examples/post_product_productid_variations_request.json:2]: yess, Maik is then using the same attributes and values

File: _raml/apps/products/examples/post_product_productid_variations_request.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            "value": "cable"
+        }
+      ]
+    },
+    {
+      "name": "wearing option",
+      "defaultForExistingProducts": "on-ear",
+      "values": [
+          {
+              "value": "in-ear"
```
Comment (dkluever22) [inline _raml/apps/products/examples/post_product_productid_variations_request.json:22]:
```suggestion
              "value": "In-ear"
```

File: _raml/apps/products/examples/post_product_productid_variations_request.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+          },
+          {
+              "value": "on-ear"
```
Comment (dkluever22) [inline _raml/apps/products/examples/post_product_productid_variations_request.json:25]:
```suggestion
              "value": "On-ear"
```

**Review by dkluever22 — APPROVED** [review]

**Pull request merged**